core/templates/dev/head/library/library.html

```diff
@@ -45,6 +45,12 @@
         </div>
 
         <div class="oppia-exp-summary-tiles-container" ng-if="!inSearchMode">
+          <img src="/images/library/banner.svg" align="center" width="100%">
```

Comment:
Might be better if the style settings were placed inside the CSS file, so the code looks nicer.